pkg/api/kptfile/v1alpha2/validation.go

```diff
@@ -147,13 +143,47 @@ func validateFnConfigPath(p string) error {
 	return nil
 }
 
+func ValidateFnConfigPath(pkgPath types.UniquePath, configPath string) (*yaml.RNode, error) {
```

Comment:
This function is called `Validate` but it returns an RNode. What does this mean? It actually does read, validate and return result.

Reply:
renamed to `getValidatedConfigPath`. I can change if there are other suggestions